README.md

```diff
@@ -1,10 +1,48 @@
 # Boutique
 
 [![GitHub package version](https://badgen.net/github/release/ExtensionEngine/boutique)](https://github.com/ExtensionEngine/boutique/releases)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/105e3679b14c4b7b9205981734e90424?branch=develop)](https://www.codacy.com/app/ExtensionEngine/boutique?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ExtensionEngine/boutique&amp;utm_campaign=Badge_Grade)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/105e3679b14c4b7b9205981734e90424?branch=develop)](https://www.codacy.com/app/ExtensionEngine/boutique?utm_source=github.com&utm_medium=referral&utm_content=ExtensionEngine/boutique&utm_campaign=Badge_Grade)
 [![Known Vulnerabilities](https://snyk.io/test/github/ExtensionEngine/boutique/develop/badge.svg)](https://snyk.io/test/github/ExtensionEngine/boutique)
 [![GitHub license](https://badgen.net/github/license/ExtensionEngine/boutique)](https://github.com/ExtensionEngine/boutique/blob/develop/LICENSE)
 [![js semistandard style](https://badgen.net/badge/code%20style/semistandard/pink)](https://github.com/Flet/semistandard)
 [![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=102)](https://github.com/ellerbrock/open-source-badge/)
 
->Under construction :construction:
+> ### :construction: Under construction :construction:
+
+LMS
+
+## :page_with_curl: Dependencies
+
+- Node.js (>= 8.11.0)
+- npm (>= 5.8.0)
+- PostgreSQL (>= 9.4)
```

Comment:
👍 But please add link/pointer to `package.json` `engines` field so folks know where does it come from.